README.md

```diff
@@ -1 +1,246 @@
-# react-weather-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [Credits](#credits-a-name"credits")
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Shopping <a name="about-project"></a>
+
+> The project is all about a simple interface for showing countries and cities and each city or country weather details, here is the open-weather-app main API that I used for fetching the locations [Open-Weather-App API](http://api.openweathermap.org/geo/1.0/direct?q=${location}&limit=5&appid={APIkey}). The webapp have 2 user interfaces.
+
+- A home page showing a list of locations.
+- A details page that shows details for the clicked city.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- ReactJs
+- Jest
+- Styled Component
+
+Click [here](https://www.loom.com/share/3977b01627ea4752b8f8de549a5260da) to watch a walk-through description, given by me.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Webpack</summary>
+  <ul>
+    <li><a href="https://webpack.js.org/guides/getting-started/#basic-setup">Webpack</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Jest</summary>
+  <ul>
+    <li><a href="https://jestjs.io/">Jest</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Develop webapp around an API**
+- **Conduct test to ensure all edge cases are covered**
+- **Create web page using given wireframe**
+- **Users can filter amount of categories displayed**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](#https://shopping-sage.vercel.app/)
```

Comment:
##### **Live version**
- [ ] According to the project requirement,  you should deploy the application using Heroku, Netlify, or GitHub Pages. Kindly address this 👍🏻 and make sure:
  1 The project is deployed correctly, and the home and detail page works as expected.
  2. A link to the deployed project is present in the README.